API/chat/tests.py

```diff
@@ -670,13 +617,9 @@ def test_delete_message_that_does_not_exist(self):
             timestamp=timestamp
         )
 
-        qstring = urllib.urlencode({
-            'id': message.id + 1
-        })
-
         response = self.client.delete(
-            reverse('chat:message', args=(self.channel.name,)),
-            qstring
+            reverse('chat:message', args=(message.id + 1,)),
+            {}
```

Comment:
Yes it is intentional. I want on DELETE and PATCH requests the only parameter to be the `id` of the message. Do you suggest another way to do that?